src/FE/src/app/pages/dashboard-page/dashboard-page.component.html

```diff
@@ -101,17 +102,17 @@ <h2>{{ selectedCountry?.name }}</h2>
         <mat-divider></mat-divider>
         <div class="infoContainer">
           <mat-list>
-            <div mat-subheader>Costs</div>
+            <div mat-subheader class="selectedCountryUndertitle">{{ 'PROJECT_PRICE' | translate }}</div>
             @for (countryData of selectedCountry!.projects; track countryData) {
               <mat-list-item>
                 <div matListItemTitle>{{ countryData.name }}</div>
                 <div
-                  matListItemLine>{{ this.currencyService.getSymbol(localStorage.getItem("selectedCurrency")!) }} {{ countryData.cost | number: '1.0-2' }}
+                  matListItemLine>{{ this.currencyService.getSymbol(localStorage.getItem("selectedCurrency")!) }} {{ countryData.price | number: '1.0-2' }}
```

Comment:
er det muligt, at dele denne i to linjer? uden det ødelægger viewet. men bare så vi ikke har kilometer lange linjer